animata/pills/scattered-pills.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { motion } from "framer-motion";
+
+const pills = [
+  { id: 1, text: "poor analytics", color: "bg-purple-400" },
+  { id: 2, text: "no strategy", color: "bg-orange-300" },
+  { id: 3, text: "running out of ideas", color: "bg-green-300" },
+  { id: 4, text: "engaging content", color: "bg-blue-300" },
+  { id: 5, text: "replying to mentions", color: "bg-red-300" },
+];
+
+export default function InteractivePillsPyramid() {
+  const [hoveredPills, setHoveredPills] = useState<number[]>([]);
```

Comment:
Actually, you don't need these states. The pills become straight when the parent div is hovered and are rotated by default.